web/gds-admin-ui/src/utils/index.js

```diff
@@ -1,6 +1,29 @@
+import config from '../config';
+import { ENVIRONMENT } from '../constants';
+
 export * from './array';
 
 
+function defaultEndpointPrefix() {
```

Comment:
Could we also write some tests for the utils in this function? I'd like to make sure we aren't creating too much technical debt for testing; but we can get to the bulk of that next week or the week after.

Reply:
alright